maintenance/wikia/sql/dataware-schema.sql

```diff
@@ -14,562 +14,1046 @@
-  `user_text` varchar(255) default NULL,
-  KEY `eye2_cookie` (`cookie`),
-  KEY `eye2_all` (`cookie`,`city_id`,`id`,`page`)
+DROP TABLE IF EXISTS `image_review_stats`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `image_review_stats` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `reviewer_id` int(11) NOT NULL,
+  `wiki_id` int(11) NOT NULL,
+  `page_id` int(11) NOT NULL,
+  `review_state` int(11) NOT NULL DEFAULT '0',
+  `review_end` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (`id`),
+  KEY `image_idx` (`wiki_id`,`page_id`),
+  KEY `state_date_idx` (`reviewer_id`,`review_state`,`review_end`)
+) ENGINE=InnoDB AUTO_INCREMENT=47723529 DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `image_review_wikis`
+--
+
+DROP TABLE IF EXISTS `image_review_wikis`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `image_review_wikis` (
+  `wiki_id` int(11) NOT NULL,
+  `top200` tinyint(1) DEFAULT NULL,
+  PRIMARY KEY (`wiki_id`),
+  KEY `top200` (`top200`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
-SET character_set_client = @saved_cs_client;
+/*!40101 SET character_set_client = @saved_cs_client */;
 
 --
--- Table structure for table `global_watchlist`
+-- Table structure for table `info_data`
 --
 
-SET @saved_cs_client     = @@character_set_client;
-SET character_set_client = utf8;
-CREATE TABLE IF NOT EXISTS `global_watchlist` (
-  `gwa_id` int(11) NOT NULL auto_increment,
-  `gwa_user_id` int(11) default NULL,
-  `gwa_city_id` int(11) default NULL,
-  `gwa_namespace` int(11) default NULL,
-  `gwa_title` varchar(255) default NULL,
-  `gwa_rev_id` int(11) default NULL,
-  `gwa_timestamp` varchar(14) default NULL,
-  PRIMARY KEY  (`gwa_id`),
-  KEY `user_id` (`gwa_user_id`)
-) ENGINE=InnoDB AUTO_INCREMENT=20813231 DEFAULT CHARSET=latin1;
-SET character_set_client = @saved_cs_client;
-
---
--- Table structure for table `magcloud_collection`
---
-
-SET @saved_cs_client     = @@character_set_client;
-SET character_set_client = utf8;
-CREATE TABLE IF NOT EXISTS `magcloud_collection` (
-  `mco_hash` varchar(32) NOT NULL,
-  `mco_user_id` int(11) default '0',
-  `mco_wiki_id` int(11) default '0',
-  `mco_updated` datetime default NULL,
-  `mco_articles` text,
-  `mco_cover` text,
-  `mco_publish` text,
-  PRIMARY KEY  (`mco_hash`)
+DROP TABLE IF EXISTS `info_data`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `info_data` (
+  `wid` int(11) DEFAULT NULL,
+  `id` int(11) DEFAULT NULL,
+  `title` tinytext,
+  `info_key` tinytext,
+  `value` text,
+  `template` tinytext,
+  `additional` text
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
-SET character_set_client = @saved_cs_client;
+/*!40101 SET character_set_client = @saved_cs_client */;
 
```

Comment:
`info_*` tables seem to be some left over after abandoned project. I'll rename then with a `__` prefix. We'll see if someone complains ;)